frontend/src/containers/Ledger.js

```diff
@@ -153,34 +153,41 @@ export class Ledger extends Component {
 
   }
 
+  loadCollectiveData() {
+    const { collective, params, fetchProfile } = this.props;
+    if (collective && collective.id) return Promise.resolve(collective);
+    return fetchProfile(params.slug).then(result => {
```

Comment:
don't need to return anything here. `fetchProfile` will automatically populate the collective data. See `fetchProfile` in `collective.js`.